pact-request-api/Pact/Core/Command/Types.hs

```diff
@@ -145,16 +145,13 @@ type SigCapability = CapToken QualifiedName PactValue
 -- | Pair parsed Pact expressions with the original text.
 data ParsedCode = ParsedCode
   { _pcCode :: !Text
-  , _pcExps :: ![Lisp.TopLevel ()]
+  , _pcExps :: ![Lisp.TopLevel Info]
```

Comment:
Why this change to `SpanInfo`?

Reply:
Good question. 

We want chainweb's choice of `Info` to be governed by the `Info` alias in `Evaluate.hs`

`ParsedCode` was nuking line information before. Now we get to keep it and be consistent.